.gitignore

```diff
@@ -118,3 +118,4 @@
 /ml-commons/
 /opensearch-benchmark/
 /opensearch-benchmark-workloads/
+/observability/
```

Comment:
remove `trace-analytics` from .gitiginore?

Reply:
had a discussion here to keep it https://github.com/opensearch-project/opensearch-plugins/pull/113#issuecomment-994162677